test/drenv/zap.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import logging
+
+# Map zap log levels to python log levels.
+# https://github.com/uber-go/zap/blob/6d482535bdd97f4d97b2f9573ac308f1cf9b574e/zapcore/level.go#L113
+_LOGGERS = {
+    "debug": logging.debug,
+    "info": logging.info,
+    "warn": logging.warn,
+    "error": logging.error,
+    "dpanic": logging.error,
+    "panic": logging.error,
+    "fatal": logging.error,
+}
+
+
+def log_json(line, name="zap"):
+    try:
+        info = json.loads(line)
+    except ValueError:
+        # We don't want to crash if another program has a logging bug, and the
+        # line may contain useful info.
+        logging.debug("[%s] %s", name, line)
+        return
```

Comment:
Discussed with @parikshithb, will move the error handler to the caller, so it can use the right log level for the context. This is good for kubectl gather (no need for warnings if we get non-json logs, but other case may need different solution.